mpv.py

```diff
@@ -479,6 +479,12 @@ def command(self, name, *args):
                 for arg in args if arg is not None ] + [None]
         _mpv_command(self.handle, (c_char_p*len(args))(*args))
 
+    def play(self):
+        self._set_property("pause", False, proptype=bool)
+
+    def pause(self):
```

Comment:
Oh I was not aware (or forgot) that you could access properties that way. I'll just remove the pause function for now and rename the other one as suggested.